examples/deploy/terraform/cluster/variables.tf

```diff
@@ -1,7 +1,79 @@
-## Used to overwrite the `k8s_version` provided at initial creation.
-## When upgrading k8s, create/modify tfvars with desired `k8s_version` value.
-variable "k8s_version" {
-  description = "Update k8s version."
-  type        = string
-  default     = null
+## Used to overwrite the `eks` variable passed through the `infra` outputs.
+
+variable "eks" {
+  description = <<EOF
+    creation_role_name = Only meant to support an imported role.
+    k8s_version = EKS cluster k8s version.
+    kubeconfig = {
+      extra_args = Optional extra args when generating kubeconfig.
+      path       = Fully qualified path name to write the kubeconfig file.
+    }
+    public_access = {
+      enabled = Enable EKS API public endpoint.
+      cidrs   = List of CIDR ranges permitted for accessing the EKS public endpoint.
+    }
+    "Custom role maps for aws auth configmap
```

Comment:
I think we're either missing the end quote or needing to change the beginning quote to a `#` (or remove it)